apps/web/src/app/layouts/toggle-width.component.ts

```diff
@@ -0,0 +1,33 @@
+import { CommonModule } from "@angular/common";
+import { Component } from "@angular/core";
+
+import { PlatformUtilsService } from "@bitwarden/common/platform/abstractions/platform-utils.service";
+import { NavigationModule } from "@bitwarden/components";
+
+@Component({
+  selector: "app-toggle-width",
+  template: `<bit-nav-item
+    text="Toggle Width"
+    icon="bwi-bug"
+    *ngIf="isDev"
+    class="tw-absolute tw-bottom-0 tw-w-full"
+    (click)="toggleWidth()"
+  ></bit-nav-item>`,
+  standalone: true,
+  imports: [CommonModule, NavigationModule],
+})
+export class ToggleWidthComponent {
+  protected isDev: boolean;
+
+  constructor(platformUtilsService: PlatformUtilsService) {
+    this.isDev = platformUtilsService.isDev();
+  }
```

Comment:
```suggestion
  protected isDev: boolean = inject(PlatformUtilsService).isDev()
```

Thoughts on favoring this syntax more generally in components?

Reply:
I'm not against it, it does make things more angular specific though, not sure how all testing tools handles this?